pkgs/development/libraries/librsvg/default.nix

```diff
@@ -15,7 +15,7 @@
 , rustPlatform
 , rustc
 , rust
-, cargo
+, cargo-auditable-cargo-wrapper
```

Comment:
Add a TODO here as well for when auditable is the default?

Reply:
Currently it only works for `buildRustPackage`, do you think it should just be a override in cargo then?

Reply:
Making it consistent (opt-out) tree-wide would be better but I guess after `buildRustPackage` is covered there isn't that much left.

Leave it for now and we can revisit this when auditable is made the default?

Reply:
sounds good